apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/HHAClaimTransformerV2Test.java

```diff
@@ -153,13 +153,58 @@ public void shouldSetFinalAction() {
   public void shouldSetBillablePeriod() throws Exception {
     // We just want to make sure it is set
     assertNotNull(eob.getBillablePeriod());
+    Extension extension =
+        eob.getBillablePeriod()
+            .getExtensionByUrl("https://bluebutton.cms.gov/resources/variables/claim_query_cd");
+    Coding valueCoding = (Coding) extension.getValue();
+    assertEquals("Final bill", valueCoding.getDisplay());
+    assertEquals("3", valueCoding.getCode());
+    assertEquals(
+        "https://bluebutton.cms.gov/resources/variables/claim_query_cd", valueCoding.getSystem());
     assertEquals(
         (new SimpleDateFormat("yyy-MM-dd")).parse("2015-06-23"),
         eob.getBillablePeriod().getStart());
     assertEquals(
         (new SimpleDateFormat("yyy-MM-dd")).parse("2015-06-23"), eob.getBillablePeriod().getEnd());
   }
 
+  /** Tests that the billable period is not set if claim query code is null. */
```

Comment:
Assuming we have a test if the query code is not null based on the test json we have?